.travis.yml

```diff
@@ -4,33 +4,42 @@ sudo: false
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION=1.4.x-dev
+    - COMPOSER_ROOT_VERSION=1.x-dev
     - CORE_RELEASE=master
 
 matrix:
   fast_finish: true
   include:
     - php: 5.6
-      env: DB=MYSQL PHPUNIT_TEST=1 PHPCS_TEST=1
+      env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.0
       env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL PDO=1 PHPUNIT_COVERAGE_TEST=1
+      env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.2
+      env: DB=MYSQL PHPUNIT_TEST=1  PHPCS_TEST=1 PHPUNIT_COVERAGE_TEST=1
```

Comment:
Having both PHPUNIT_TEST and PHPUNIT_COVERAGE_TEST will run the test suite twice, you can remove PHPUNIT_TEST